script.js

```diff
@@ -0,0 +1,11 @@
+const navLink = document.querySelectorAll('.nav-link');
+const menuButton = document.querySelector('.menu-button');
+const navbarLinks = document.querySelector('.navbar-links');
+function navSlide() {
+  navbarLinks.classList.toggle('active');
+}
+function close() {
+  navbarLinks.classList.remove('active');
+}
+menuButton.addEventListener('click', navSlide);
+navLink.forEach((n) => n.addEventListener('click', close));
```

Comment:
- [ ] **[OPTIONAL]:** I'm sorry that the previous reviewer didn't point this out, but there's a requirement that is not yet met. 
![image](https://user-images.githubusercontent.com/77069011/203549884-bb75a6af-b815-4e52-9d6d-656b3fee63c5.png)
This could be due to a typo in your `navLink` variable, you're selecting all the elements with the `nav-link` class, but the links have `nav-links` class. Please consider fixing this as it improves tremendously navigating through your page and user experience! 🤓 🛠️